HLTrigger/Configuration/python/HLT_75e33/modules/hltGeneralTracks_cfi.py

```diff
@@ -24,3 +24,58 @@
     trackAlgoPriorityOrder = cms.string('trackAlgoPriorityOrder'),
     writeOnlyTrkQuals = cms.bool(False)
 )
+
+_hltGeneralTracksSingleIterPatatrack = hltGeneralTracks.clone(
+    TrackProducers = ["hltInitialStepTrackSelectionHighPurity"],
+    hasSelector = [0],
+    indivShareFrac = [1.0],
+    selectedTrackQuals = ["hltInitialStepTrackSelectionHighPurity"],
+    setsToMerge = [cms.PSet(
+        pQual = cms.bool(True),
+        tLists = cms.vint32(0)
+    )]
+)
+
+from Configuration.ProcessModifiers.singleIterPatatrack_cff import singleIterPatatrack
+singleIterPatatrack.toReplaceWith(hltGeneralTracks, _hltGeneralTracksSingleIterPatatrack)
+
+_hltGeneralTracksLST = hltGeneralTracks.clone(
+    TrackProducers = ["hltInitialStepTrackSelectionHighPuritypTTCLST", "hltInitialStepTrackSelectionHighPuritypLSTCLST", "hltInitialStepTracksT5TCLST", "hltHighPtTripletStepTrackSelectionHighPurity"],
```

Comment:
as a generic note, I'm proposing to monitor the intermediate track collection produced in the HLT menu at https://github.com/cms-sw/cmssw/pull/46823. 
I suppose we should customize those developments once this PR enters.

Reply:
I agree and, in fact, I have already done this while I was preparing the PR: please check [this commit](https://github.com/SegmentLinking/cmssw/pull/94/commits/8cae22c7f716025c3fa76e936a64637687e458bd). Especially given the presence of https://github.com/cms-sw/cmssw/pull/46823, I thought I would include it in a next PR (after rebasing) but let me know whether it makes sense to include it already and whether you have an comments on the implementation.

Reply:
> I thought I would include it in a next PR (after rebasing) but let me know whether it makes sense to include it already and whether you have an comments on the implementation.

let's do it as a follow-up step (incidentally, I think we need to change the event content of the step in which the HLT runs in order to make the collections available for the step when the DQM is run).

Reply:
Indeed, in my tests I was manually keeping the extra collections in the `cmsDriver` command. Let's follow up with this after this gets merged.